app/views/layouts/application.html.haml

```diff
@@ -26,7 +26,7 @@
     |
     %a{:href => '#nav'} Skip to Page Navigation
 
-  = render :partial => 'shared/test_banner'
```

Comment:
Is there a print.css in this project?

Reply:
If so I would just add a display: none; rule to the banner in there.
Otherwise perhaps throw in the banner.scss file:
@media print {
  display: none;
}

?